test/integration/generated/url/Source/PathItems.swift

```diff
@@ -300,8 +300,8 @@ public final class PathItems {
         if let localStringQuery = options?.localStringQuery {
             queryParams.append("localStringQuery", localStringQuery)
         }
-        if let globalStringQuery = client.globalStringQuery {
-            queryParams.append("globalStringQuery", globalStringQuery)
+        if let GlobalStringQuery = client.GlobalStringQuery {
```

Comment:
i think one of the changes from 'serializedName' to 'name' break this. The name should be 'globalStringQuery' instead of captical 'GlobalStringQuery'